src/components/app/AppViewRouter.js

```diff
@@ -25,15 +25,16 @@ import { assertExhaustiveCheck } from 'firefox-profiler/utils/flow';
 import type { AppViewState, State, DataSource } from 'firefox-profiler/types';
 
 import type { ConnectedProps } from 'firefox-profiler/utils/connect';
+import { Localized } from '@fluent/react';
 
-const ERROR_MESSAGES: { [string]: string } = Object.freeze({
-  'from-addon': "Couldn't retrieve the profile from Firefox.",
-  unpublished: "Couldn't retrieve the profile from Firefox.",
-  'from-file': "Couldn't read the file or parse the profile in it.",
-  local: 'Not implemented yet.',
-  public: 'Could not download the profile.',
-  'from-url': 'Could not download the profile.',
-  compare: 'Could not retrieve the profile',
+const ERROR_MESSAGES_L10N_ID: { [string]: string } = Object.freeze({
+  'from-addon': 'AppViewRouter--error-message-from-addon',
+  unpublished: 'AppViewRouter--error-message-unpublished',
```

Comment:
Probably these 2 strings should be the same (probably the "unpublished", because in the future we'd like to replace `from-addon` by `unpublished`). I'll do the change myself.